css/_vertical_filmstrip_overrides.scss

```diff
@@ -103,6 +103,20 @@
                 right: 150px;
             }
         }
+
+        &.with-filmstrip.without-remote-videos {
```

Comment:
I've added a longer comment above the styles. I think some cleanup can be done with the styles too but maybe in the near future as filmstrip work continues, just in case this breaks during the process.